text_encoder.go

```diff
@@ -106,6 +110,42 @@ func (enc *textEncoder) AddMarshaler(key string, obj LogMarshaler) error {
 	return err
 }
 
+// Helpers for Add Array fns below
+func (enc *textEncoder) addArrayBegin() {
+	enc.bytes = append(enc.bytes, '[')
+}
+
+func (enc *textEncoder) addArrayEnd() {
+	enc.bytes = append(enc.bytes, ']')
+}
+
+func (enc *textEncoder) addArraySep(i int) {
+	if i == 0 {
+		return
+	}
+	enc.bytes = append(enc.bytes, ' ')
+}
+
+func (enc *textEncoder) AddInts(key string, vals []int) {
+	enc.addKey(key)
+	enc.addArrayBegin()
+	for i, val := range vals {
+		enc.addArraySep(i)
+		enc.addInt64(int64(val))
+	}
+	enc.addArrayEnd()
+}
+
+func (enc *textEncoder) AddStrings(key string, vals []string) {
+	enc.addKey(key)
+	enc.addArrayBegin()
```

Comment:
Don't think it's worth it, since we're also chasing #189 - there's a good chance that we're going to pivot the text encoder to use the JSON encoder.